packages/react-components/src/components/Tooltip/Info.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Icon, IconTypeName } from '../Icon';
+import { Close } from '@livechat/design-system-icons/react/material';
+import { getIconType } from './helpers';
+import styles from './Tooltip.module.scss';
+
+const baseClass = 'lc-tooltip';
+
+export const Info: React.FC<{
+  header: string;
+  text: string;
+  closeWithX?: boolean;
+  theme?: string;
+  handleCloseOnClick?: () => void;
+}> = ({ header, text, closeWithX, theme, handleCloseOnClick }) => {
+  return (
+    <div style={{ position: 'relative' }}>
+      {closeWithX && (
+        <div className={styles[`${baseClass}-x`]} onClick={handleCloseOnClick}>
```

Comment:
IMO this class should be named `${baseClass}-close`. WDYT?

Reply:
👌🏼